tracks/python/exercises/pig-latin/pig-latin/mentoring.md

```diff
@@ -0,0 +1,42 @@
+# Pig Latin
+
+
+## Common Areas of Improvement
+When mentoring students on the Pig Latin exercise, it's essential to focus on common areas where they might struggle. Here are some key points to keep in mind:
+
+1. Understanding the Rules Clearly
```

Comment:
This doesn't have anything to do wit Pig Latin. Mentoring typically occurs after the solution passes the tests. If the tests pass, presumably the solution is implemented correctly and all edge cases have been accounted for.

I would recommend mentoring this exercise a couple of times then reviewing some common themes that come up in the mentoring discussions. I wouldn't expect that mentors need to call out edge cases when the tests already enforce those.